general-superstaq/general_superstaq/service_test.py

```diff
@@ -58,6 +58,12 @@ def test_update_user_balance(
     )
 
 
+def test_update_user_balance_limit() -> None:  # pylint: disable=missing-function-docstring
```

Comment:
I think this pragma can be removed cause we already ignore docstring for test files.
```suggestion
def test_update_user_balance_limit() -> None:
```